src/context/auth-context.tsx

```
@@ -31,6 +31,19 @@ export const AuthProvider = ({ children }: { children: React.ReactNode }) => {
   };
 
+  const register = async (data: RegisterData) => {
+    try {
+      const { name, email, password } = data;
+      await api.post("/auth/register", {
+        name,
+        email,
+        password,
+      });
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)